packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts

```diff
@@ -7,6 +7,8 @@ import { SimpleEnabledTrade, Solver } from '@scripts/libraries/types';
 import * as wallet from '@test-utils/wallet';
 import { ethers } from 'hardhat';
 
+const DUST_THRESHOLD = utils.parseEther('1');
```

Comment:
not sure if this should be part of the `Solver` interface so developers are forced to set a threshold which makes sense. 1 ether in 3crv is 1.02 usd.

Reply:
I mean, the thing is: what should the heuristic be here ? Profitability with respects of % of fees won by Yearn by harvest?

Reply:
Clearly the 1 ether amount is just a placerholder rn.

Reply:
something to tackle in another issue or pr for sure. :+1: